src/InputStepper/index.js

```diff
@@ -1,153 +1,141 @@
-// @flow
+// @flow strict
 import * as React from "react";
 
 import { SIZE_OPTIONS } from "../InputField/consts";
 import KEY_CODE_MAP from "../common/keyMaps";
 import InputStepperStateless from "./InputStepperStateless";
 import validateIncrement from "../utils/validateIncrement";
 import validateDecrement from "../utils/validateDecrement";
-
-import type { Props, State, ForwardedRef } from "./index";
-
-class InputStepper extends React.Component<Props & ForwardedRef, State> {
-  state = {
-    value: this.props.defaultValue || 0,
-  };
-
-  setValueAndInjectCallback = (value: number) => {
-    const { onChange } = this.props;
-    if (onChange) {
-      onChange(value);
-    }
-    this.setState({ value });
+import useStateWithCallback from "../hooks/useStateWithCallback";
+import type { Ref } from "../common/common.js.flow";
+
+import type { Props } from "./index";
+
+const InputStepper = React.forwardRef<Props, HTMLInputElement>((
+  {
+    onChange,
+    minValue = Number.NEGATIVE_INFINITY,
+    defaultValue = 0,
+    maxValue = Number.POSITIVE_INFINITY,
+    step = 1,
+    ...props
+  }: Props,
+  // $FlowExpected
```

Comment:
What's the problem with the `Ref` type? Can you fix that properly instead of suppressing it with comment? 🙏

Reply:
nope. Could not find solution. Just try to do same with other components where is suppressed comment - it will be same. No idea how to fix.